WordPress/Classes/Services/PostService.m

```diff
@@ -700,6 +697,23 @@ - (NSDictionary *)dictionaryWithKey:(NSString *)key inMetadata:(NSArray *)metada
     return [matchingEntries lastObject];
 }
 
++ (NSString *)keyForType:(PostServiceType)postType {
+    NSString *key;
+    switch (postType) {
+        case PostServiceTypePage:
+            key = @"page";
+            break;
+        case PostServiceTypePost:
+            key = @"post";
+            break;
+        case PostServiceTypeAny:
+        default:
+            key = @"any";
+            break;
+    }
+    return key;
+}
```

Comment:
I think I'll rewrite `PostService` in swift next, and then `+ keyForType:` can be removed, and handled by the enum directly 🎉